aepp/hygiene.py

```diff
@@ -188,7 +188,7 @@ def deleteDatasetExpiration(self,ttlId:str=None)->dict:
         path = f"/ttl/{ttlId}"
         if self.loggingEnabled:
             self.logger.debug(f"Starting deleteDatasetExpiration, ttlId: {ttlId}")
-        res = self.connector.deleteData(self.endpoint+path)
+        res = self.connector.postData(self.endpoint+path)
```

Comment:
Not the method you want to change I believe ;) 
But thanks for spotting the error on the `createRecordDeleteRequest`